pkg/oc/bootstrap/docker/up.go

```diff
@@ -1040,6 +1094,10 @@ func useAnsible(v semver.Version) bool {
 	return v.GTE(openshiftVersion36)
 }
 
+func useWebConsoleTemplate(v semver.Version) bool {
+	return v.GTE(openshiftVersion39)
```

Comment:
you can use v.GT(openshiftVersion37) instead to avoid introducing yet another version number constant.

Reply:
It's not clear to me looking at the doc if that will return true for 3.7.1?

https://godoc.org/github.com/blang/semver#Version.GT

I might need to make this greater than v3.9.0-alpha.0 to avoid the extended test failures.

Reply:
it should return true for 3.7.1.

Reply:
> it should return true for 3.7.1.

Cluster up will fail if it tries to use the web console image for 3.7.1 since the image doesn't exist.

Reply:
ok well all the "current/previous" logic is impacted by that.  so it should all be updated to check for 3.9.